src/deploy/project.py

```diff
@@ -214,6 +214,31 @@ def _get_service(service):
 
         return matched_services
 
+
+    def confirm_service_deployment(self, release, environment_id):
+        ecs_services = self.get_ecs_services(release, environment_id)
+
+        deployed_services = []
+        for image_id, services in sorted(ecs_services.items()):
+            for service in services:
+                tags = service["response"]["tags"]
+                service_deployment_label = next(tag["value"] for tag in tags if tag["key"] == "deployment:label")
+                ecs = self._ecs(
+                    region_name=service.get('region_name'),
+                    role_arn=service.get('role_arn')
+                )
+                tasks = ecs.list_service_tasks(service)
+
+                deployed_tasks = []
+                for task in tasks:
```

Comment:
You might want to check if the number of tasks is equal to the number of desired tasks in order to get a definitive "good" deployment.

Reply:
Great point, done.